Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># CSE250_Project2_product_Enquiry_quotation_\r\nThis repository is for the project of the course CSE250 DBMS. \r\n\r\ntesing th\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision a09a26e748aea5d0d6bfb62e8573ccd247eb076f)
+++ b/README.md	(date 1770216185531)
@@ -1,4 +1,3 @@
 # CSE250_Project2_product_Enquiry_quotation_
 This repository is for the project of the course CSE250 DBMS. 
 
-tesing th
